src/variables.jl

```diff
@@ -0,0 +1,259 @@
+################################################################################
+#                              LOGICAL VARIABLES
+################################################################################
+# helper function to create the variable info used when creating reformulation variables
+function _variable_info(;
+    binary::Bool=false, 
+    lower_bound::Float64=-Inf, upper_bound::Float64=Inf, 
+    start_value::Union{Nothing, Bool}=nothing, 
+    fix_value::Union{Nothing, Bool}=nothing
+)
+    JuMP.VariableInfo(
+        !isinf(lower_bound), lower_bound, 
+        !isinf(upper_bound), upper_bound, 
+        !isnothing(fix_value), fix_value, #fix value
+        !isnothing(start_value), start_value, #start value
+        binary, false #integrality
+    )
+end
```

Comment:
I know I said differently before, but I think we should remove this and just create the reformulation variables using `@variable`.

Reply:
done